src/platform/imx8/memory.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 NXP
+ *
```

Comment:
new file, need a license !

I now see the link with the SPDX question, now we need to agree on that *before* merging this code